### DISCUSSION TITLE
Quick search like Codespace or Alfred just for massCode

### DISCUSSION THREAD

**Discussion by pkd2989** [massCodeIO/massCode#106] [Ideas]
Hi, 
I use dual boot setup, Windows + Linux 
Currently I use https://codespace.app/ for storing my snippets, which is supported on both platform.

Pros: 
The main feature that let me keep using this is QuickSearch, wherever I am I can do "Alt+S" and start typing and it will filter the snippet, and with a press of Enter, it is copied to my clipboard. And hence I can use it with anycode editor/IDE and both OS.

Cons: 
The only shortcoming with Codespace app is that it doesn't have a shared Folder, which I can setup and access it from other OS. The only way is to Export the collection and import it on another. 

Oppurtunity/Idea:  For massCode, just add the similar QuickSearch feature as this will be very much useful. I will switch full time to massCode if this is implemented. 

Thanks for work.
Regards

**Comment by antonreshetov** [comment]
Hi @pkd2989

In the application you can press "Cmd+F" or "Ctrl+F" to focus on the search field. Perhaps that's what you're looking for.  

For macOS users, there is integration with [Raycast](https://www.raycast.com/antonreshetov/masscode) and [Alfred](https://github.com/massCodeIO/assistant-alfred). For all platforms there is integration with [VS Code](https://marketplace.visualstudio.com/items?itemName=AntonReshetov.masscode-assistant).

**Comment by pkd2989** [comment]
Hi, Thanks for the reply.
No.
I meant, I should be able to do snippet search without having to launch the massCode in front. Like in Codespace, I can search and select snippet without having to leave my working editor/IDE. The Codespace application is launched and minimized in background in system tray, and user can continue to write code, press "Alt+S" and a search dialog pops-up and I can search and copy snippet just like that. I hope I could make this clear now.

Regards,

**Reply by rinormaloku** [reply]
I didn't know about Codespace, looks exactly what I need, but doesn't run on Ubuntu 22.04

**Reply by pkd2989** [reply]
It have it working on Ubuntu 22.04.

**Reply by rinormaloku** [reply]
This is the error that I am getting:
```
23:22:56.677 › APPIMAGE env is not defined, current application is not an AppImage
[542451:0714/232257.212722:FATAL:gpu_data_manager_impl_private.cc(986)] The display compositor is frequently crashing. Goodbye.
[1]    542451 trace trap (core dumped)  codespace
```

**Reply by pkd2989** [reply]
Sorry, you are correct. 
I had Ubuntu 20.04 LTS, where it was working. With 22.04 it is crashing. I have written to the developer.

Also there are no updates since long. That's why having that feature on massCode may be very useful.
